test/ng/directive/ngClassSpec.js

```diff
@@ -629,7 +645,7 @@ describe('ngClass', function() {
     }));
 
     it('should not be copied when using one-time binding', inject(function($compile, $rootScope) {
-      element = $compile('<div ng-class="::{foo: veryLargeObj, bar: bar}"></div>')($rootScope);
+      element = $compile('<div ng-class="::{foo: !!veryLargeObj, bar: bar}"></div>')($rootScope);
```

Comment:
This is the change I'd like to avoid in 1.6.x by reverting the ng-class half of https://github.com/angular/angular.js/commit/189461f9bf6fda18ddbd16c42f2e959cf939c3da in 1.6